doc/source/style.rst

```diff
@@ -0,0 +1,25 @@
+.. _style:
+
+.. currentmodule:: pandas
+
+.. ipython:: python
+   :suppress:
+
+   import numpy as np
+   import pandas as pd
+   np.set_printoptions(precision=4, suppress=True)
+   pd.options.display.max_rows = 15
+
+*******
+Styling
+*******
+
+.. versionadded:: 0.17.1
+
+
```

Comment:
obviously want to do a pass on this to update (and mark experimental), but can do a follow up PR (and of course need a nice section in the whatsnew as well).